recipes/ncurses/ncurses.inc

```diff
@@ -37,6 +37,8 @@ EXTRA_OECONF += "\
 --without-cxx-binding \
 --with-terminfo-dirs=${sysconfdir}/terminfo:${datadir}/terminfo \
 --enable-overwrite \
+--enable-pc-files \
+--with-pkg-config-libdir=${libdir}/pkgconfig \
```

Comment:
it only takes a short look at the ncurses configure script to verify this is a special-kind-of-child upstream :) they seem to think it is better to implement everything with their own configure handles

so yes, ncurses is special here